### DISCUSSION TITLE
Question about MSDIAL based on Linux

### DISCUSSION THREAD

**Discussion by ArtemChemist1** [systemsomicslab/MsdialWorkbench#118] [Q&A]
Good day, I performed analysis in MsDial in command line on Ubuntu server, but I got wrong files after data analysis. It should be file with extension .mdproject. I got files with this extension .msdial. I attached my screen, method file which I used for data analysis, probably I made a mistake in this file and also screen with error cvParam. I would be very grateful for any help. Thank you in advance.

![Wrong output file](https://github.com/systemsomicslab/MsdialWorkbench/assets/128925137/1dc371b2-01da-4466-9fcc-72dee229f97f)

[Msdial_lcms_dda_Param.txt](https://github.com/systemsomicslab/MsdialWorkbench/files/11767881/Msdial_lcms_dda_Param.txt)

![cvParam](https://github.com/systemsomicslab/MsdialWorkbench/assets/128925137/8b11b2e7-a374-4ab2-aa00-1a3775dae65f)

**Comment by kozo2** [comment]
Hi @ArtemChemist1 

This may be difficult to determine its cause without having the data shared.
Could the data be made public?

**Reply by ArtemChemist1** [reply]
Dear kozo2, yes sure. Unfortunately, I can't upload data indicated above. I attached the another one parameter file and link included
mzml files for data analysis. https://drive.google.com/drive/folders/1Yn3zdmN6khBAi_NCOOiefEt866QwGV0w?usp=sharing 
[Msdial_lcms_dda_Param_testfromGNPS.txt](https://github.com/systemsomicslab/MsdialWorkbench/files/11884518/Msdial_lcms_dda_Param_testfromGNPS.txt)
Thank you in advance.

Yours Sincerely

**Reply by kozo2** [reply]
@ArtemChemist1 Thanks for sharing the data.
First we will try to reproduce your error.

**Reply by kozo2** [reply]
@ArtemChemist1 I can reproduce your problem with the MsdialConsoleApp.exe for Windows, though not for Linux.
![image](https://github.com/systemsomicslab/MsdialWorkbench/assets/12192/1c70a842-3537-4f66-8848-a800655029ef)
Please give me some time to look into it.

**Reply by ArtemChemist1** [reply]
Dear kozo2, thanks I look forward.

Yours Sincerely

**Reply by htsugawa** [reply]
Hi, 

I write the following things to make this problem clear. 
1. The command line tool is currently available at MS-DIAL 4 package.
2. The file extension of project file in MS-DIAL 4 is "*.mtd2". The "*.msdial" file just stores peak picking results by the ASCII format.
3. The "*.mtd2" format file can be generated in the command line tool of MS-DIAL 4 by the option of "-p".
4. The file extension of project file in MS-DIAL 5 is "*.mdproject".

For mzML parser issue, I have to check it at my side. Give me one week to find the time to resolve this issue.

Hiroshi

**Reply by ArtemChemist1** [reply]
Dear Dr. Hiroshi Tsugawa, thanks I look forward. Let's hope everything works out.

Yours Sincerely

**Reply by ArtemChemist1** [reply]
Dear Dr. Hiroshu Tsugawa, would you mind telling me did you manage to resolve issue with mzML parser? Thank you in advance.

Yours Sincerely

**Reply by htsugawa** [reply]
Yes, I have already fixed this issue, and you can see the update in the current version.

**Comment by kozo2** [comment]
Hi @ArtemChemist1 

Could you try http://prime.psc.riken.jp/compms/msdial/download/repository/Linux/MSDIAL%20ver.4.9.221218%20Linuxx64.zip ?

By the way, we have confirmed that it does not work in Ubuntu version 22.04. (And resolving https://github.com/systemsomicslab/MsdialWorkbench/issues/200 doesn't seem to be easy...)

If your environment is Ubuntu 20.04, it should be able to process mzML files without any problems. (Unfortunately, we don't have the time to test this on other Linux environments at the moment...)

**Reply by ArtemChemist1** [reply]
Good day, thank a lot. Yes, we have Ubuntu 20.04 version and I will try.

Yours Sincerely
